enderio-machines/src/main/java/com/enderio/machines/data/recipes/WeatherChangeRecipeProvider.java

```diff
@@ -0,0 +1,45 @@
+package com.enderio.machines.data.recipes;
+
+import com.enderio.base.api.EnderIO;
+import com.enderio.base.common.init.EIOFluids;
+import com.enderio.machines.common.blocks.obelisks.weather.WeatherChangeRecipe;
+import java.util.concurrent.CompletableFuture;
+import net.minecraft.core.HolderLookup;
+import net.minecraft.data.PackOutput;
+import net.minecraft.data.recipes.RecipeOutput;
+import net.minecraft.data.recipes.RecipeProvider;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.world.level.material.Fluid;
+import net.neoforged.neoforge.fluids.FluidStack;
+
+public class WeatherChangeRecipeProvider extends RecipeProvider {
+
+    public WeatherChangeRecipeProvider(PackOutput output, CompletableFuture<HolderLookup.Provider> registries) {
+        super(output, registries);
+    }
+
+    @Override
+    protected void buildRecipes(RecipeOutput recipeOutput) {
+        // build(EnderIO.loc("day"), EIOFluids.LIQUID_SUNSHINE.getSource(), 1000,
```

Comment:
Honestly, should we let players change the time of day? Could be abused on multiplayer servers etc, and time isn't weather